R/theme_fmt.R

```diff
@@ -25,7 +25,11 @@ fmt_combine <- function(expression, ..., missing = NULL){
 
 
 
-fmt_str <- function(group = NULL, label = NULL, fmt = NULL){
+fmt_str <- function(group = NULL, label = NULL, ...){
+  fmt <- list(...)
+  if(length(fmt) > 1){
```

Comment:
Do we have plans to accommodate multiple params in 1 fmt_str()? for example 
```
fmt_str(group = ".default", label = ".default", n = fmt("XXX"), pct = fmt("xx.x%"))
```

Reply:
At the moment no, because I think it will be confusing on the user end. But, we could, there isn't a technical reason not to. I am probably going to leave it as 1 now, just cause it is clear that fmt_str only ever applies one format either of format combine or just a normal format. If people don't find this confusing and find it more annoying to re-specify the group and label then we can change it